client.go

```diff
@@ -136,7 +136,12 @@ const (
 )
 
 func currentTimestamp() int64 {
-	return int64(time.Nanosecond) * time.Now().UnixNano() / int64(time.Millisecond)
+	return FormatTimestamp(time.Now())
+}
+
+// FormatTimestamp formats a time into Unix timestamp in milliseconds, as requested by Binance.
+func FormatTimestamp(t time.Time) int64 {
```

Comment:
This small helper facilitates to pass the correct values to Start and End params. Sadly Go doesn't format timestamp in milliseconds, it either uses Nanoseconds or Seconds. So a formatter is helpful for the consumer (unless you change Start/End to accept a Time, which would be possible but it will require significant non-BC changes).